docs/troubleshooting/README.md

```diff
@@ -59,8 +59,6 @@ The other resolution (that scales but takes some time) that was found is as foll
 
 If this problem occurs with Core lambdas and you are using the `terraform-aws-cumulus.zip` file source distributed in our release, we recommend using the non-scaling approach as the number of lambdas we distribute is in the low teens, which are likely to be easier and faster to reconfigure one-by-one compared to editing our configs.
 
-[Discussed in the Earthdata Wiki](https://wiki.earthdata.nasa.gov/display/CUMULUS/KMS+Exception%3A+AccessDeniedException).
```

Comment:
These wiki pages essentially repeat what is in these docs already or are actually more confusing/out of date. Plus they're not accessible so non-DAAC users. So it seemed best to remove them